src/bereikbaarheid/bollards/bollards.py

```diff
@@ -0,0 +1,237 @@
+from django.db import connection
+
+from bereikbaarheid.utils import django_query_db
+
+"""
```

Comment:
Is dit bedoelt als docstring voor de module, of als comment over de query? Als het eerste dan moet hij bovenaan de file (boven de imports) en anders moet het een comment zijn en geen multiline string. Ik zou zeggen dat dit het laatste is.